app/main.py

```diff
@@ -1,3 +1,67 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers"
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(
+                km=self.km + other.km
+            )
```

Comment:
Bad code style, you have alone closing Parentheses and not alone the opening one. Either make keys starting on a new line or move the closing one to the preceding line. Also you should reform many next methods of your class under previous rule.